rllib/evaluation/rollout_worker.py

```diff
@@ -1601,8 +1601,14 @@ def get_state(self) -> dict:
         """
         filters = self.get_filters(flush_after=True)
         policy_states = {}
-        for pid in self.policy_map:
-            policy_states[pid] = self.policy_map[pid].get_state()
+        for pid in self.policy_map.keys():
+            if (
```

Comment:
cool. can you add a comment saying we are checking if users only want to checkpoint trainable policies here.
thanks a ton!

Reply:
Oh, sorry, yes will add this!